src/cryptonote_protocol/quorumnet.cpp

```diff
@@ -1704,6 +1711,8 @@ void setup_endpoints(cryptonote::core& core, void* obj) {
             // Receives blink tx signatures or rejections between quorum members (either original or
             // forwarded).  These are propagated by the receiver if new
             .add_command("blink_sign", [&qnet](Message& m) { handle_blink_signature(m, qnet); })
+            // Receives a request for the timestamp
+            .add_command("timestamp", [&qnet](Message& m) { handle_timestamp(m, qnet); })
```

Comment:
I think you can drop the `qnet` argument and capture here: we don't have any actual state to track for this timestamp command.